saleor/graphql/webhook/enums.py

```diff
@@ -102,6 +120,22 @@
     WebhookEventAsyncType.VOUCHER_DELETED: "A voucher is deleted.",
     WebhookEventAsyncType.ANY: "All the events.",
     WebhookEventAsyncType.OBSERVABILITY: "An observability event is created.",
+    WebhookEventSyncType.SHIPPING_LIST_METHODS_FOR_CHECKOUT: (
+        "Fetching external shipping methods for checkout."
```

Comment:
We can use the same form for all sync events:
```suggestion
        "Fetch external shipping methods for checkout."
```